build.gradle

```
@@ -39,6 +39,15 @@ version = '1.0.0-SNAPSHOT'
 description = 'tc-guide-working-with-jooq-flyway-using-testcontainers-gradle'
 java.sourceCompatibility = 17
 
+def jooqGeneratedDir = layout.getBuildDirectory().dir("generated-sources/jooq").get().asFile.path
+sourceSets {
+    main {
+        java {
+            srcDirs += [jooqGeneratedDir]
+        }
+    }
+}
+
```

This can be removed as of jooq 3.19.10
```suggestion
def jooqGeneratedDir = layout.getBuildDirectory().dir("generated-sources/jooq").get().asFile.path
sourceSets {
    main {
        java {
            srcDirs += [jooqGeneratedDir]
        }
    }
}

```